From fd6676eb16f79374da6a7d8eed48de46741daaa6 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 05 Jun 2022 16:10:48 +0300
Subject: [PATCH] Revert "[SW-77251] gaudi3: add global register init"

Revert rom recovery implementation for secured firmware update via
inband and OOB.

This reverts commit ab9e934b63ef3b986525c57bc27a3c805d79c870.

Change-Id: I95b5087ffe4c7482b0743689cf7dc1c75be5e4d6
---

diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index c765289..1095a99 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -17,7 +17,6 @@
 #include <dev_efuse.h>
 #include <dev_dma.h>
 #include <device.h>
-#include <gaudi3_reg_map.h>
 
 static struct cmd_module_ops cmd_ops[] = {
 	{	/* PCI module */
@@ -143,24 +142,11 @@
 	ext_rom_load_erom_image(HL_FW_EROM_RECOVERY);
 }
 
-static void device_init_global_conf(void)
-{
-	global_conf.upd_cmd_map = mmUPD_CMD;
-	global_conf.upd_sts_map = mmUPD_STS;
-}
-
-static void device_init(void)
-{
-	device_init_global_conf();
-}
-
 static void rom_recovery(void)
 {
 	/* ROM recovery not supported in slave die */
 	if (arcboot_rom_global.die_id)
 		return;
-
-	device_init();
 
 	module_handle_cmds(cmd_ops, ARRAY_SIZE(cmd_ops));
 }
